.github/workflows/tensorflow.yml

```diff
@@ -31,6 +31,8 @@ jobs:
         pip install git+https://github.com/optuna/optuna-integration.git
         python -c 'import optuna_integration'
 
+        # NOTE(nabenabe0928): Got "AttributeError: module 'tensorflow' has no attribute 'estimator'".
```

Comment:
I think it would be better to replace this comment as a TODO comment to remove the version constraint.

Reply:
Done!